notification/src/main/java/com/chronica/notification/data/dto/AlertDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.chronica.notification.data.dto;
+
+import com.chronica.notification.data.constant.PriorityType;
+import com.chronica.notification.data.dto.abstraction.BaseDTO;
+import com.chronica.notification.data.dto.integrant.NotificationDTO;
+import lombok.*;
+
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class AlertDTO extends BaseDTO {
+    private PriorityType priorityType;
+    private NotificationDTO baseData;
```

Comment:
Powinno extendować NotificationDTO, ponieważ Alert to rozszerzony notification. Tak by było gdyby Notification było polem w Alercie